app/src/main/java/com/vero/tasky/agenda/presentation/taskdetails/TaskDetailsEvent.kt

```diff
@@ -0,0 +1,16 @@
+package com.vero.tasky.agenda.presentation.taskdetails
+
+import com.vero.tasky.agenda.presentation.model.ReminderRange
+import java.time.LocalDate
+import java.time.LocalTime
+
+sealed interface TaskDetailsEvent {
+    object ChangeMode: TaskDetailsEvent
+    class CheckModifiedInfo(val title: String?,
```

Comment:
What's the use if this event?

Reply:
The same as in EventScreen, if the user changes the title or description I need to pass them from navBackEntry to the screen, and then from LauncedEffect to viewModel (and for this I'm using this event)

Reply:
Ahh gotcha